src/parse.jl

```diff
@@ -29,7 +29,7 @@ function tryparsenext_fixedtz(str, i, len, min_width::Int=1, max_width::Int=0)
         return Nullable{String}(), i
     else
         tz = SubString(str, tz_start, tz_end)
-        return Nullable{String}(tz), i
+        return Nullable{String}(convert(String, tz)), i
```

Comment:
Hmm, this is explicitly required? Can we get away with `Nullable{String}(String(tz))`?

Reply:
Yup